framework/tests/base_test.py

```diff
@@ -4,21 +4,24 @@
 from selenium.webdriver.support.ui import WebDriverWait
 from browser.py_quality_services import PyQualityServices
 from core.utilities.json_settings_file import JsonSettingsFile
+
 from framework.forms.main_form import MainForm
+from framework.utils.browser_factory import BrowserFactory
 
 
 class Test_base:
 
-    @pytest.fixture(autouse = True)
-    def setup(self):
-        self.settings = JsonSettingsFile("config.json")
+
+    @pytest.fixture(scope="session", autouse=True)
+    def prepare_browser_factory(request):
```

Comment:
this method currently has nothing to do with browser factory
please remove this one. you should have fixture methods only in conftest.py